WordCounter/WordCounter.cs

```diff
@@ -8,22 +8,63 @@ namespace WordCounter
 {
     class WordCounter
     {
-        public WordCounter(string inputString)
+        private Dictionary<string, int> dict;
+
+        public WordCounter()
         {
-            Dictionary<string, int> dict = new Dictionary<string, int>();
+            dict = new Dictionary<string, int>();
+        }
 
-            char[] delimiters = { ' ', ',', '.' };
+        public void Start(string inputString)
+        {
+            //Note: some ? marks are still escaping somehow
```

Comment:
This is due to the file being in UTF8 format while Windows and C# default to UTF16.  In .net Core 3.0 we will get the ability to enumerate over runes in a string (yes similar to golang).